mysql-test/suite/innodb/t/percona_session_temp_tablespaces_encrypt.test

```diff
@@ -0,0 +1,109 @@
+let $MYSQLD_DATADIR=`SELECT @@datadir`;
+
+# Using on disk tablespace for intrinsic optimiser tables
+SELECT @@innodb_temp_tablespace_encrypt;
+SET GLOBAL big_tables=ON;
+
+--echo # Create a new connection.
+connect (con1, localhost, root,,);
+
+connection con1;
+
+--echo # A session tablespace will be allocated to connection 1
```

Comment:
Later: if the testcase stability issues are changing connection IDs, a server restart at the start would make them deterministic sequence. The same might apply to x in temp_x.ibt too?

Reply:
Perfect suggestion! restart solved the issue